tests/components/conftest.py

```diff
@@ -25,7 +25,7 @@ def prevent_io():
 
 
 @pytest.fixture
-def entity_registry_enabled_by_default() -> Generator[AsyncMock, None, None]:
+def entity_registry_enabled_by_default() -> Generator[MagicMock, None, None]:
```

Comment:
Since no integration makes use of the return value, we could probably adjust this to be `Generator[None, None, None]`
I've converted it for now to `MagicMock` as it's a dataclass property: nothing Async about it.

Reply:
Let's make it a `None` indeed